graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -235,7 +235,7 @@ def rename_schema(
     schema = build_ast_schema(schema_ast)
     query_type = get_query_type_name(schema)
 
-    _validate_renamings(schema_ast, type_renamings, field_renamings, query_type)
+    _ensure_no_unsupported_suppressions(schema_ast, type_renamings)
```

Comment:
After this PR, the only separate validation step left would be this one, so there's no need for a special `_validate_renamings` function.